src/swagger-specs/destinations.yaml

```diff
@@ -34,81 +34,313 @@ servers:
+      <table><thead><tr><th>Term</th><th>Description</th></tr></thead>
+      <tbody>
+      <tr><td>Base connection</td><td>Base connections retain authentication information regarding how to connect to a target destination. You must set up a base connection before you can create a target connection.</td></tr>
+      <tr><td>Source connection</td><td>Source connections create and manage a connection to the internal source from where data is exported. For destination connections, the source connection is always the Experience Platform Profile Store.</td></tr>
+      <tr><td>Target connection</td><td>Target connections create and manage a destination connection to any location where exported data will land. Target connections contain information regarding data destination, data format, and the target connection ID needed to create a destination dataflow.</td></tr>
+      <tr><td>Dataflow</td><td>A dataflow is a virtual pipeline of data that flows into Platform from a source and out to a destination. This API reference focuses on destination dataflows.</td></tr>
+      <tr><td>Dataflow run</td><td>A dataflow run is an execution instance of a dataflow that lands in Experience Platform based on a user-specified schedule OR that is exported from Experience Platform to a destination of your choice, also based on a user-specified schedule and other criteria. This API reference focuses on dataflow runs to destinations.</td></tr>
+      </tbody></table>     
+  - name: Base connections
+    description: >
+          Base connections retain information regarding how to connect to a source or target. 
+          For destinations, the source that you are connecting to is Experience Platform data and the target is the desired destination. 
+          The base connection configuration always includes a unique `connectionSpec ID`, which is necessary in order to successfully access Experience Platform data and authorize a connection to the target destination.
+  - name: Source connections
+    description: Source connections create and manage a connection to the internal source from where data is exported. For destination connections, the source connection is always the Experience Platform Profile Store. Source connections contain information like data source, data format, and the source connection ID needed to create a dataflow. A source connection instance is specific to a tenant and IMS Organization.
+  - name: Target connections
+    description: Target connections create and manage a destination connection to any location where exported data will land. Target connections contain information regarding data destination, data format, and the target connection ID needed to create a destination dataflow. A target connection instance is specific to a tenant and IMS Organization. <br><br> Before you create a target connection, you <i>must</i> create a base connection.
```

Comment:
Few mentions of "IMS Organization" which need to be updated to just "organization"